src/containers/Form/FormHeader.jsx

```diff
@@ -6,29 +6,130 @@
  *
  */
 
-import React from 'react';
+import React, { Fragment } from 'react';
```

Comment:
OBS: Her vil det komme en endring mtp. på formik. Da må dette også inn for FormikHeader.